schemas/server.yml

```diff
@@ -74,3 +74,10 @@
       example: 12
       description: >
         Packets sent from the server to the client.
+
+    - name: uptime
```

Comment:
Sorry I hadn't noticed this. The field set `server` is meant to descrive the server side of a network connection. We have two pairs: client/server and source/destination. As such, this isn't the right place for this.

`server.uptime` should be moved to `host.uptime` :-)

Let's also make `host.uptime` extended for now.

Reply:
Is `extended` a matter of adding a `level: extended`, or is there any other consideration?